<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="np-sadekunle" name="Default test" tests="5" failures="1" timestamp="17 Jun 2015 08:49:58 GMT" time="19.213" errors="0">
  <testcase name="DepositTest" time="18.101" classname="Cashier.DepositTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:210)
	at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:94)
	at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:70)
	at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:44)
	at Cashier.DepositTest.DepositTest(DepositTest.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- DepositTest -->
  <testcase name="DepositTest" time="0.094" classname="Cashier.DepositTest">
    <skipped/>
  </testcase> <!-- DepositTest -->
  <testcase name="DepositTest" time="0.09" classname="Cashier.DepositTest">
    <skipped/>
  </testcase> <!-- DepositTest -->
  <testcase name="DepositTest" time="0.092" classname="Cashier.DepositTest">
    <skipped/>
  </testcase> <!-- DepositTest -->
  <testcase name="DepositTest" time="0.095" classname="Cashier.DepositTest">
    <skipped/>
  </testcase> <!-- DepositTest -->
</testsuite> <!-- Default test -->
